### ISSUE TITLE
SVG Parser: Use Dimensions Of Input Graph

### ISSUE THREAD

**Issue by Ian-Stewart-Binks** [Courseography/courseography#551]
The SVG parser currently has the dimensions of the output graph hardcoded in. This should be taken from the input graph.

Note that this likely means that the database will need to know this information.

**Comment by david-yz-liu** [comment]
This is correct, although we should probably also specify some "standard size" to act as a guideline.

**Comment by Ian-Stewart-Binks** [comment]
Something along the lines of having upper bounds on the dimensions?

We will need to coordinate this with the drawing tool, as well.

**Comment by david-yz-liu** [comment]
So, a checklist:

- [ ] Update the Wiki graph-creation instructions to have the users check for a maximum document size
- [ ] Update SVG parser to get the width and height attributes of the SVG file
- [ ] Update the SVG Generator so that it uses the custom width and height for each graph
